queries/python/indents.scm

```diff
@@ -20,6 +16,19 @@
   (concatenated_string)
 ] @indent
 
+((list) @aligned_indent
+ (#set! "delimiter" "[]")
+ (#set! "dedent_bare_closing_delim" 1)
+)
+((dictionary) @aligned_indent
+ (#set! "delimiter" "{}")
```

Comment:
A note for future. Each delimiter should be provided separately (as separate args) and not as one string.

Reply:
@georgeharker I think this could be separated into setting opening and closing delimiter rn, and could be done in this PR. The change would only require switching 5LOC

Reply:
agreed, pushed that change.

Reply:
I am leaving wholesale changes to the way delimiters are specified out of this PR - it makes it larger and should be done with a clearer understanding of how that should match the way injections work.

Reply:
If the sentiment is strong to have them split, it's easy to do so.  But I can do so as a clear separate commit to view just those diffs, with the others squashed.